config/cl-authorization/config.lisp

```diff
@@ -0,0 +1,135 @@
+;;;;;;;;;;;;;;;;;;;
+;;; delta messenger
+(in-package :delta-messenger)
+
+(setf *delta-handlers* nil)
+(add-delta-logger)
+(add-delta-messenger "http://deltanotifier/")
+
+;;;;;;;;;;;;;;;;;
+;;; configuration
+(in-package :client)
+(setf *log-sparql-query-roundtrip* t)
+(setf *backend* "http://triplestore:8890/sparql")
+
+;;;;;;;;;;;;;;;;;
+;;; access rights
+
+(in-package :acl)
+
+(defparameter *access-specifications* nil
+  "All known ACCESS specifications.")
+
+(defparameter *graphs* nil
+  "All known GRAPH-SPECIFICATION instances.")
+
+(defparameter *rights* nil
+  "All known GRANT instances connecting ACCESS-SPECIFICATION to GRAPH.")
+
+(type-cache::add-type-for-prefix "http://mu.semte.ch/sessions/" "http://mu.semte.ch/vocabularies/session/Session")
+
+(define-prefixes
+  :lblodmow "http://data.lblod.info/vocabularies/mobiliteit/"
+  :dct "http://purl.org/dc/terms/"
+  :skos "http://www.w3.org/2004/02/skos/core#"
+  :org "http://www.w3.org/ns/org#"
+  :mobiliteit "https://data.vlaanderen.be/ns/mobiliteit#"
+  :vs "http://www.w3.org/2003/06/sw-vocab-status/ns#"
+  :ext "http://mu.semte.ch/vocabularies/ext/"
+  :cidoc "http://www.cidoc-crm.org/cidoc-crm/"
+  :qudt "http://qudt.org/schema/qudt/"
+  :foaf "http://xmlns.com/foaf/0.1/"
+  :nfo "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#"
+  :rdfs "http://www.w3.org/2000/01/rdf-schema#"
+  :sh "http://www.w3.org/ns/shacl#"
+  :tribont "https://w3id.org/tribont/core#"
+  :musession "http://mu.semte.ch/vocabularies/session/"
+  :variables "http://lblod.data.gift/vocabularies/variables/"
+)
+
+(define-graph public-graph ("http://mu.semte.ch/graphs/public")
```

Comment:
Until now, the public resources were also stored on the `http://mu.semte.ch/graphs/mow/registry` graph. The change in this config would mean that users which are not logged in have no read access to the `http://mu.semte.ch/graphs/mow/registry` graph.